src/components/EventLocationFilter/EventLocationFilter.js

```diff
@@ -49,59 +52,98 @@ class EventLocationFilter extends Component {
     this.validateLocation(this.state.location);
   }
 
-  handleOnChangeLocation(e) {
-    // If user has entered exactly 5 digits, check to see if they're valid
-    // this avoids user having to trigger input blur to have field validated
-    const location = e.target.value;
+  getGeolocation() {
+    if (window.navigator.geolocation) {
+      window.navigator.geolocation.getCurrentPosition((pos) => {
+        const path = `?latlng=${pos.coords.latitude},${pos.coords.longitude}&sensor=true`;
+        axios.get(`http://maps.googleapis.com/maps/api/geocode/json${path}`).then((res) => {
```

Comment:
As a matter of separation concerns it seems like we should only be making API calls from an API/Service module and not use `axios` in a component file.